code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -823,21 +847,24 @@ GLOBAL_LIST_EMPTY(PDAs)
 		cartridge.host_pda = src
 		to_chat(user, "<span class='notice'>You insert [cartridge] into [src].</span>")
 		update_icon()
-		playsound(src, 'sound/machines/button.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/button.ogg', 15, 1)

```

Comment:
Sound is already quiet, volume change can be reverted. Also, this is a mechanical sound, making this mutable via a software toggle makes no logical sense